plugins/micromega/coq_micromega.ml

```diff
@@ -402,7 +404,7 @@ struct
     * pp_*      functions pretty-print Coq terms.
     *)
 
-  exception ParseError
+  exception ParseError of string option
```

Comment:
Why the `option`? When there is no useful explanation for the error, you may resort to an empty string.

Reply:
done 3b8c3263fb